supereeg/brain.py

```diff
@@ -366,8 +381,8 @@ def resample(self, resample_rate=None):
             self.data = data
             self.sessions = sessions
             self.sample_rate = sample_rate
-            if not sample_rate == resample_rate:
-                self.kurtosis = _kurt_vals(self)
+            #if not sample_rate == resample_rate:
```

Comment:
I think we need to re-compute `self.dur` here (could just add a line with `self.update_info()`

Reply:
also: remove extraneous comments around these lines